### DISCUSSION TITLE
Filters inside a frontmatter

### DISCUSSION THREAD

**Discussion by eshlox** [maizzle/framework#962] [Q&A] [answered]
Is it possible to use filters inside a frontmatter?

The previous version has a [Transform Content](https://v3.maizzle.com/docs/transform/) feature that I use to translate emails. Example:

```md
---
title: <raw i18n>Title</raw>
---
```

I've migrated from `transform` to `filters`, but it does not work the same way. Was this change intentional? Can I somehow modify the frontmatter content?

I may try to write a custom posthtml plugin, but maybe there is an easier way.

**Comment by cossssmin** [comment, marked as answer]
PostHTML tags don't work inside Front Matter, but you can use a [function](https://maizzle.com/docs/functions) instead:

```js
// config.js
module.exports = {
  // ...
  uppercase: string => string.toUpperCase()
}
```

```xml
---
title: "{{{ page.uppercase('Some title') }}}"
---
```

Notice how the variable must be wrapped in double quotes `""`.

You can also nest the `uppercase` function in a `locals` object if you don't like to call if off the `page` object:

```js
// config.js
module.exports = {
  // ...
  locals: {
    uppercase: string => string.toUpperCase(),
  },
}
```

```diff
---
- title: "{{{ page.uppercase('Some title') }}}"
+ title: "{{{ uppercase('Some title') }}}"
---
```
